boards/arm/frdm_k64f/Kconfig.defconfig

```diff
@@ -98,6 +98,13 @@ config SPI_0
 
 endif # SPI
 
+if CHARACTER_FRAMEBUFFER
```

Comment:
@jfischer-phytec-iot  I have to put it here because the CHARACTER_FRAMEBUFFER_DISPLAY_DEV_NAME is defined in the subsys/fb/Kconfig, and according to the config rules marcos in samples/display/Kconfig can not override those defined in subsys/fb/Kconfig, and this change is requested by @vanwinkeljan. And user can override this in the sample/display/cfb/Kconfig once we have this defined in board default config.  Actually the CFB driver need to find a supported display, and this selection currently is statically controlled by marco, unless we have a new framwork to replace. but this could be in another patch set.

Reply:
@hakehuang I did not realize that the board had no display but if this the case I agree with @jfischer-phytec-iot that it should not be put in the boards defconfig. You could move it into the board specific Kconfig overlay of the sampels (eg. `samples/display/cfb_shell/boards/frdm_k64f.conf`)

Reply:
@vanwinkeljan , @jfischer-phytec-iot , can I ask for your help on which marco shall I use, if Ido not create one in the board Kconfig, I need to use a existing one, but none of the display driver export their driver name with a marco.

Reply:
I would just add `CHARACTER_FRAMEBUFFER_DISPLAY_DEV_NAME="SSD1306"` to `samples/display/cfb_shell/boards/frdm_k64f.conf` for now, in the end it is the DTS overlay (`samples/display/cfb_shell/frdm_k64f.overlay`) in the sample that sets the name.

Reply:
I suggest to move all `CHARACTER_FRAMEBUFFER_DISPLAY_DEV_NAME="display"` from  _Kconfig.defconfig_ to samples/display/cfb_shell/boards/_board_.conf. It does not really solve the problem, but not have that much impact.

Reply:
I see. Thanks a lot.

Reply:
> Can I make this way?

No as stated before this will make it a big mess as for each display you need to add a new default entry. The error message clearly states the issue, the Kconfig symbol `CHARACTER_FRAMEBUFFER_DISPLAY_DEV_NAME ` has no prompt.

Could you try by adding a prompt (`string  "Character Framebuffer Display Driver Name"`)?

Reply:
ok, thanks this works